spec/features/index_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe 'index page' do
+  it 'shows a food' do
+    visit '/'
+
+    fill_in 'q', with: 'sweet potato'
+    click_on 'Search'
+
+    expect(current_path).to eq('/foods')
+    expect(page).to have_content('Sweet potato')
+    expect(page).to have_content('Total items')
+  end
```

Comment:
This looks like it was on the right track, but if I'm following the interaction pattern exactly then I want to setup the test to match exactly. So, an expectation for the actual number of total items, and a UPC code, and the description, etc. Generally, I would set up as much of the test as possible up front, so that I can use the error messages to guide my implementation.